Makefile

```diff
@@ -62,7 +62,7 @@ HAXE_VERSION_SHORT=$(shell echo "$(HAXE_VERSION)" | grep -oE "^[0-9]+\.[0-9]+\.[
 ifneq ($(STATICLINK),0)
 	LIB_PARAMS= -cclib '-Wl,-Bstatic -lpcre -lz -Wl,-Bdynamic '
 else
-	LIB_PARAMS?= -cclib -lpcre -cclib -lz
+	LIB_PARAMS?= -cclib -lpcre -cclib -lz -cclib
```

Comment:
Shouldn't it be `-cclib -lcrypt` or something? (builds with that)

Reply:
Err... you're right. Now I'm confused why it builds. I was getting linker errors before...